components/post_view/post/post.jsx

```diff
@@ -111,10 +113,14 @@ export default class Post extends React.PureComponent {
 
     componentDidMount() {
         this.postRef.current.addEventListener(A11yCustomEventTypes.ACTIVATE, this.handleA11yActivateEvent);
+        document.addEventListener('keydown', this.handleAlt);
```

Comment:
@Willyfrog I think this needs to be handled in the A11 controller, @devinbinnie can you confirm?

Reply:
I don't think it needs to be, both listeners are at the `document` level so they should both fire. I don't think anything should interfere with how the handler here works, but @deanwhillier might have more information.

Reply:
@enahum @deanwhillier do you feel we need to change it here?

Reply:
A11Y controller uses the capture phase of events so regular bubble event listeners like these don't prevent it from receiving user events.

Reply:
Even if `event.preventDefault()` is used.